src/store/modules/util/index.ts

```diff
@@ -15,7 +15,9 @@ const utilModule: Module<UtilState, RootState> = {
     partyNames: {},
     cancelReasons: [],
     facilities: {},
-    enumerations: {}
+    enumerations: {},
+    currentFacilityInformation: {},
+    storesInformation: {}
```

Comment:
```suggestion
    storesInformation: []
```

Update the default value to array as we are storing an array in the state.